integration-tests/manifests/cmd/validate_instrumentation_vars.go

```diff
@@ -18,11 +18,21 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+var appSignalsEnvVarKeys = []string{
+	"OTEL_AWS_APP_SIGNALS_ENABLED",
+	"OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+	"OTEL_TRACES_SAMPLER_ARG",
+	"OTEL_TRACES_SAMPLER",
+	"OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+	"OTEL_AWS_APP_SIGNALS_EXPORTER_ENDPOINT",
+}
+
 func main() {
 
 	args := os.Args
 	namespace := args[1]
 	jsonFilePath := args[2]
+	appSignals := args[3] == "app_signals"
```

Comment:
so appSginals is a boolean flag? can we renamed it to something like `appSignalsEnabled` or something?